django-backend/fecfiler/committee_accounts/views.py

```diff
@@ -220,3 +220,19 @@ def remove_member(self, request, pk: UUID):
         member = self.get_object()
         member.delete()
         return HttpResponse("Member removed")
+
+    def update(self, request, *args, **kwargs):
+        committee_uuid = request.session["committee_uuid"]
+        role = (
+            Membership.objects.filter(
+                user=request.user, committee_account_id=committee_uuid
+            )
+            .values_list("role", flat=True)
+            .first()
+        )
+        if role != Membership.CommitteeRole.COMMITTEE_ADMINISTRATOR:
+            return Response(
+                "You don't have permission to perform this action.",
+                status=status.HTTP_403_FORBIDDEN,
```

Comment:
Because DRF returns 403 when a user is not authenticated with a valid sessionid, we may want to use 401 here so that the user doesn't get logged out/redirected to the login page https://github.com/fecgov/fecfile-web-app/blob/9ca387bced84d83335556bb03b11dbb420b3879b/front-end/src/app/shared/interceptors/http-error.interceptor.ts#L41